### PR TITLE
add html and css files

### PR DESCRIPTION
**Pull request by mirelapitt**
This is where I'll past the html and css files to be review from the project that I did on the trials of the Microverse Program.

### CODE REVIEW

**Review by mirelapitt — COMMENTED** [review]
hi @mirelapitt !

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

**Highlights**

- The repo does not look professional ❌
- Git/GitHub flow best practices does not have been followed ❌
- There are no linter installed ❌

**Required Changes** ♻️

- [ ] Following GitHub best practices, you should write meaningful commit messages to document your project professionally. It is not easy to change commit history, I only ask you to correct this behavior in the future.

- [ ] For every project, it is required to install the linters, to make sure that there's no linter errors and your code looks professional.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+       
+
+<div class="container"><header class="header">
```
Comment (mirelapitt) [inline index.html:13]: - [ ] I recommend using the section tag for the main container that contains all of the cards and using the **div** tag for each individual card.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+      type="text"
+      name="name"
+      placeholder="Enter your name"
+      required
+    </div>
+
+    <div class="form-group">
+      <label id="email-label" for ="email">Email</label>
+      <br>
+      <input
```
Comment (mirelapitt) [inline index.html:32]: - [ ] Kindly add a list tag to the form part in order to place the inputs in a better way.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    <div class="form-group">
+      <label id="age-label" for ="age">Age</label>
+      <br>
+      <input
+      type="age"
+      name="age"
+      placeholder="Age"
+      required
+    </div>
+
```
Comment (mirelapitt) [inline index.html:48]: - [ ] All of its attributes are optional, but it's standard practice to always set at least the **[action]** and the **[method]** to the forms.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+      <br>
+      <input
+      type="describe"
+      name="describe"
+      placeholder="Select Current Role"
+      required
+    </div>
+
+
+<p>Would you recommend freeCodeCamp to a friend?</p>
```
Comment (mirelapitt) [inline index.html:60]: - [ ] I suggest you kindly add the **hx** tag and not a **p** tag in this context.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @mirelapitt, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- You are suggesting changes kindly, well done!
- You are giving a 'why' in your review, great!

## Required Changes ♻️

- [x] Great job giving your first review! It would be great if you tried an inline comment too, this is a nice way to suggest a change and point out the part of the code you are talking about.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+                               <br>
+                          <label><input
+                           name="prefer"
+                           value="Videos"
+                           type="checkbox"
+                           class="input-checkbox"
+                    
+                               />Videos</label>
+
```
Comment (mirelapitt) [inline index.html:181]: - [ ] Kindly format your code in a better way to look more professionally

**Review by mirelapitt — COMMENTED** [review]
**Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @mirelapitt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**